src/app/test_executive/hard_fork.ml

```diff
@@ -17,6 +18,75 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
 
   type dsl = Dsl.t
 
+  module Balance = struct
```

Comment:
Maybe `Balances` would be better, so this module is not confused with `Currency.Balance`.

Reply:
Maybe, although the intention was for this module to replace `Currency.Balance` for this test. In fact the balances used here are not of type `Currency.Balance.t` at all, but rather a triplet explicitly stating the liquid and locked balance. But I'm not opposed to calling it otherwise if you think that's better.

Reply:
Yes, I think `Balances` helps indicate that triple.